### PR TITLE
Add PHP8 support, also bump phpunit version

### PR DESCRIPTION
**Pull request by zlodes**
... And set min php version to 7.2 (phpunit 8 requirement)

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,7 +1,6 @@
 language: php
 
 php:
-  - 7.1
   - 7.2
```
Comment (Ocramius) [inline .travis.yml:4]: I think `- 7.4` and eventually `- 7.3` are needed. Honestly could go with just 7.4 and 8.0 BTW

Comment (zlodes) [reply .travis.yml]: @Ocramius I agree with you. Updated with `7.4` and `8.0` versions.

**Comment by Ocramius** [comment]
Good 'ol travis-ci is gonna die when I merge this :-)

**Comment by zlodes** [comment]
@Ocramius I'll fix it. Maybe tomorrow. 🙄

**Comment by Ocramius** [comment]
Travis is my problem space: don't worry. Same as you, I also need to plan this change in my schedule

**Comment by Ocramius** [comment]
Manually merged this into #35, thanks @zlodes!

**Pull request closed without merging**